rfd/0108-agent-census.md

```diff
@@ -1,6 +1,6 @@
 ---
 authors: Vitor Enes (vitor@goteleport.com)
-state: draft
+state: implemented
```

Comment:
If this has been released, can you also mark the release it was introduced in?

Reply:
The teleport backports have not been released yet. I'll open a follow-up PR once they are.

Reply:
If they're already merged, you can just assume they'll be in the next release too.

Reply:
Done in a4292a59c107183fba183e0ab8adb850c4aea7be. It assumes that the next releases for v11 and v12 will be patch releases.